input.js

```diff
@@ -30,10 +30,15 @@ function fillStars(value) {
 
 //Load new movie title into input.html based on index.html
 const movieTitleEl = document.querySelector("#enter-movie-title"); //Element in input.html for the movie title
-let newTitle = readLocalStorageNewTitle();
-
-movieTitleEl.setAttribute('value', newTitle[(newTitle.length - 1)].newMovieTitle); //automatically set the movie title to the last movie title you stored in local storage from index.html
+function loadMovieTitle() {
+    let newTitle = readLocalStorageNewTitle();
```

Comment:
This one will also need to be defined somewhere so that we have something to call